src/fare-contracts/components/InspectionSymbol.tsx

```diff
@@ -27,8 +26,6 @@ export type InspectionSymbolProps = {
 
 export const InspectionSymbol = ({
   preassignedFareProduct,
-  fromTariffZone,
-  toTariffZone,
```

Comment:
Can also remove these from the `InspectionSymbolProps`

Reply:
Done